components/Cards/index.js

```diff
@@ -17,3 +17,122 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+
+// const addCard = document.querySelector('.cards-container');
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles') 
+.then( (response) =>{
+    // console.log(response.data.articles);
+
+    // const qsa = (i) => document.querySelector(i);
+    // console.log('This is qsa', qsa);
+
+    // const grabBootstrap = qsa(response.data)
+    // console.log('This is grabBootstrap', grabBootstrap);
+
```

Comment:
You could also store response.data.articles inside a variable array and loop over the variable array with a forEach method.